RELEASE.md

```diff
@@ -60,11 +54,12 @@ Commit your changes and push them to the `release` branch when you are done.
 
 ## Update Documentation
 
-There are a couple of references in the [docs](https://github.com/quintilesims/layer0/tree/release/docs-src/docs) section that will need to be updated with the latest version:
+After running `make update-release` this will update a version number for these files and the 
```

Comment:
[KISS](https://en.wikipedia.org/wiki/KISS_principle)!!

We already have a release process that, aside from `git` housekeeping, works with a single command: 
```
make release
```
Let's try to keep it that way unless we absolutely can't.